### ISSUE TITLE
Unify protocol error reporting

### ISSUE THREAD

**Issue by sagamusix** [miranda-ng/miranda-ng#846]
Examples: The MSN protocol may report
"Unrecognised error 999. The server has closed our connection [Yes] [No]"
With two buttons which from my observation do exactly the same (i.e. it really should be just one [OK] button).
ICQ also uses message boxes for reporting connection errors.
Jabber on the other hand reports no connection errors at all.

Display of reported errors really should be unified between protocols. For example, I'm not really interested by message boxes from MSN/ICQ stacking up, so it would be nice to just globally turn off connection error messages, or to log them all into the same window, etc.

**Comment by Robyer** [comment]
These message boxes are usually only fallback when user doesn't use Popup+/YAPP plugins.
Try to use one of these plugins for popup notifications, then you will get better user experience and usually also ability to disable specific popups (if the plugin emitting popups supports it explicitly in own options, or uses popup classes - in that case you can manage them in Popup+/YAPP options.

**Comment by sagamusix** [comment]
Good idea but it doesn't seem to work. I activated Popup+ (including NewEventNotify) but I keep getting the MSN error messages.
